geth/rpc/call_raw.go

```diff
@@ -20,10 +22,21 @@ const (
 // thus, we will use zero ID as a workaround of this limitation
 var defaultMsgID = json.RawMessage(`0`)
 
+var externalRPCMethodWhitelist = []string{
```

Comment:
@mandrigin are these really all JSON-RPC methods we should allow? INFURA, for example, supports many more `eth_*`.

Reply:
Those are the rpc methods that were referenced by #912 (personal_sign and personal_ecRecover) plus the rpc methods that needed to be added to pass the mailbox tests (web3_clientVersion and eth_syncing). Where can I look or who can I ask for a more complete listing of the methods that need to be added here? Once I build the app with these changes, are there any good example apps I can run to check if I'm breaking anything with this limited whitelist?

Reply:
Supported methods by INFURA may be a good initial list. Please take a look at [`route.go`](https://github.com/status-im/status-go/blob/develop/geth/rpc/route.go#L35-L88).

> Once I build the app with these changes, are there any good example apps I can run to check if I'm breaking anything with this limited whitelist?

@mandrigin or people in `#core-dapps` channel should be able to help with this.

Reply:
@adambabik everything else is exported anyway, full namespaces, as normal. So far we really need `personal_sign` and `personal_ecRecover`, but we should not expose the whole `personal_*` namespace.
That was done implicitly, and I'm glad that this PR makes it explicit! :)